test/services/EnglishUIBuilderSpec.scala

```diff
@@ -937,6 +953,38 @@ class EnglishUIBuilderSpec extends BaseSpec with ProcessJson with EnglishLanguag
       }
     }
 
+    "Convert a empty warning list into a WarningText" in new WarningTest {
```

Comment:
I'd move these to their own test section to simplify the tests (like Details and Date Input)